Packs/QRCodeReader/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### QR Code Reader - goqr.me
+- Removed unnecessary package imports
+- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
```

Comment:
Make sure to change the name of this file to `1_0_3.md`

Reply:
done